dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationResourceTest.java

```diff
@@ -585,11 +607,114 @@ public void Test_Required_Params() throws IOException, DotDataException, DotSecu
         }
     }
 
+    @Test
+    public void Test_Pagination_And_Sort_Then_Request_Filter_Expect_Empty_Results() {
 
+        final Map<String, Param> paramMap = ImmutableMap.of(
+                "param1", Param.newParam("val-1", false, Type.STRING, "label", "hint", true),
+                "param2", Param.newParam("val-2", false, Type.STRING, "label", "hint", true),
+                "param3", Param.newParam("val-3", false, Type.STRING, "label", "hint", true)
+        );
+        final long timeMark = System.currentTimeMillis();
+        final List<Host> hosts= new ArrayList<>();
+        final char[] alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".toCharArray();
+        for(char chr :alphabet) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'chr' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4960082)